Add rabies/lassa/YFV blog post [#1051]

genehack opened the pull request:
## Description of proposed changes

Add rabies/lassa/yellow fever blog post 

[Preview of post](https://nextstrain-s-rabies-las-qbdbtb.herokuapp.com/blog/2025-02-21-rabies-lassa-yellow-fever-resources)

## Related issue(s)

Closes #1051

trvrb commented:
Cool! It would be nice if we had slightly better CSS for tables. The current use looks overly minimal:

<img width="727" alt="table" src="https://github.com/user-attachments/assets/f9d95ce1-bb59-4284-b47e-087007fe9250" />

Here's what I did for bedford.io:

<img width="937" alt="Screenshot 2025-02-20 at 11 37 07 AM" src="https://github.com/user-attachments/assets/5943d70a-a876-437d-ae7d-76de373d385b" />

Example from [this blog post](https://bedford.io/blog/reproducible-peer-review/)

<img width="750" alt="Screenshot 2025-02-20 at 11 39 06 AM" src="https://github.com/user-attachments/assets/3696801e-3975-4f5d-8099-4ea5c2a46ff8" />

From https://bedford.io/guide/style/

This CSS is from Bootstrap: https://github.com/blab/blotter/blob/master/css/custom.scss#L254

genehack commented:
> Cool! It would be nice if we had slightly better CSS for tables. 

yeah, i was not overly happy with that specific aspect either. i looked at the prior art in the past blog posts and only found one other table being used, which is why i just threw this over the wall as-is. 

> From https://bedford.io/guide/style/
> 
> This CSS is from Bootstrap: https://github.com/blab/blotter/blob/master/css/custom.scss#L254

i will see what i can do, either with this or some other styling.

@trvrb in your opinion is this a "block this blog post until the table formatting is sorted" thing, or is this a "table formatting as a fast-follow update and the blog post goes out initially as-is"? (i lean towards the latter, but i realize i'm also very much in "get the blog post done" headspace right now…)

genehack commented:
> Cool! It would be nice if we had slightly better CSS for tables.

PR has been updated.

The version of bootstrap that is embedded in site is pretty out-of-date. Rather than either updating it or trying to dig out how Bootstrap tables _used_ to work, I extracted relevant styling from the bedford.io site and added it specifically to the CSS module for blog posts. 

The table in this post now looks like this:

<img width="918" alt="Screenshot 2025-02-20 at 17 50 41" src="https://github.com/user-attachments/assets/c9bbc5ff-d7b9-4122-8f7f-8ec8765c7440" />

The table in the [one other post containing a table](https://next.nextstrain.org/blog/2019-10-21-auspice-v2) now looks like this:

<img width="955" alt="Screenshot 2025-02-20 at 17 50 27" src="https://github.com/user-attachments/assets/9fbb7cd1-a4b6-4152-921d-c2582cd5d4b6" />

I think this is sufficient unto the current need; I will plan on merging and then promoting this tomorrow, to be followed by social media posts on Monday morning, so as to not miss folks over the weekend.

genehack commented:
Pushed one additional change updating the blog publication date to Feb 21.

genehack commented:
@j23414 and @kimandrews, since your names are on this too, would be good if you could give the deployed preview version a quick skim of your post section and make sure I didn't mess up any of the links or introduce any typos or mis-formatting in the conversion process.

Thanks!

trvrb commented:
Thanks for cleaning up the table CSS. We may want to do a general CSS pass at some point (I'm assuming we'll run into similar things with `<pre>` and `<quote` and so forth), but this is just fine for blog post.

genehack commented:
> We may want to do a general CSS pass at some point (I'm assuming we'll run into similar things with `\<pre\>` and `<quote` and so forth)

TBPH, at least in my head, that's part-and-parcel with "rebuild the blog along more standard lines and possibly migrate it to a dedicated hostname/URL".
